.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: go
+
+go:
+  - 1.6
+  - 1.7.x
+  - 1.8.x
+  - 1.9.x
```

Comment:
As simple as this? Coool!

However it seems Travis-CI fails with 1.6 and 1.7.x versions.